docs/includes/mailbox-object-desc.md

```diff
@@ -0,0 +1,17 @@
+Outlook add-ins primarily use a subset of the API exposed through the [Mailbox](/javascript/api/outlook/Office.mailbox) object. To access the objects and members specifically for use in Outlook add-ins, such as the [Item](/javascript/api/outlook/Office.mailbox) object, you use the [mailbox](/javascript/api/outlook/Office.mailbox) property of the **Context** object to access the **Mailbox** object, as shown in the following line of code.
+
+```js
+// Access the Item object.
+var item = Office.context.mailbox.item;
+
+```
+
+Additionally, Outlook add-ins can use the following objects:
+
+-  **Office** object: for initialization.
+
+-  **Context** object: for access to content and display language properties.
+
+-  **RoamingSettings** object: for saving Outlook add-in-specific custom settings to the user's mailbox where the add-in is installed.
```

Comment:
I didn't originally write this content (it was copied/moved as-is from another article into this include file), so I'm not 100% sure of the rationale for including this here. I'd suspect it's because the way you save settings in an Outlook add-in is different than the way you do so for other types of add-ins (i.e., by using other objects in the Common API) -- so the author wanted to call special attention to `RoamingSettings` here.